Sources/SwiftFormatConfiguration/Configuration.swift

```diff
@@ -120,6 +120,12 @@ public struct Configuration: Codable, Equatable {
     self.version = highestSupportedConfigurationVersion
   }
 
+  /// Constructs a Configuration using values from a JSON config file
+  public init(configFile url: URL) throws {
```

Comment:
To match the naming convention of types like `Data` that load URLs, let's call this `init(contentsOf url: URL) throws`.